test/e2e/utils/helpers.go

```diff
@@ -285,24 +286,7 @@ func GetStatefulSetsForScyllaCluster(ctx context.Context, client appv1client.App
 }
 
 func GetPodsForStatefulSet(ctx context.Context, client corev1client.CoreV1Interface, sts *appsv1.StatefulSet) (map[string]*corev1.Pod, error) {
```

Comment:
this function is now a trivial wrapper - why not delete it altogether?

Reply:
The intention was:
- not duplicating the code between v1 and v1alpha1 utils
- not having to modify all existing tests for v1
- tests being able to use a single import (v1 utils xor v1alpha1 utils)